savant/deepstream/pyfunc.py

```diff
@@ -21,6 +22,7 @@ class NvDsPyFuncPlugin(BasePyFuncPlugin):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._sources = SourceInfoRegistry()
+        self.frame_streams = []
```

Comment:
I'm not sure we need cv2.cuda.Stream in any pyfunc. May be better to have it only in bgremover for now? @bwsw

Reply:
@denisvmedyantsev 100% disagree. Why the user would not want to handle `GpuMat` in arbitrary PyFunc?